pkg/mgrconfig/load.go

```diff
@@ -237,6 +237,10 @@ func (cfg *Config) completeServices() error {
 func (cfg *Config) initTimeouts() {
 	slowdown := 1
 	switch {
+	case cfg.Type == "qemu" && runtime.GOARCH == "arm64" && cfg.SysTarget.Arch == "arm64":
+		// ARM64 hosts often don't support nested virtualization, in which case KVM acceleration is unavailable.
```

Comment:
Qemu on normal host don't need nested virtualization.
Can we grep the VM config for "-accel tcg" or something similar?

Reply:
AFAIK `-accel tcg` won't be very helpful here -- I did not see it significantly speed up emulation in our case. If we grep the options, it's better to grep for `kvm`.

Reply:
`kvm` is default in this case and won't be listed. `-accel tcg` does not speedup execution, it slows it down (that's what we have on our instances). I suggested to use x5 slowdown when `-accel tcg` is present.

Reply:
If `-accel tcg` only slows it down, why do we need it?

Reply:
I am also confused by "-accel tcg does not speedup execution, it slows it down ". Or did you mean enabling it instead of `-accel kvm` is slower?

Reply:
If host arch matches target arch, then we assume that it will run native speed with KVM acceleration, and use slowdown=1 (no slowdown).

The case you want to handle is our syzbot instance that does not use KVM virtualization even that it has matching host/target arch. The instance has "-accel tcg" in the config to disable use of KVM. Because of use "-accel tcg" it is slower than normal instances with matching host/target arch that will use KVM by default in such case.

The check you added makes things better for TCG case, but makes this worse for KVM case. So I proposed to check for "-accel tcg" and set slowdown=5 only in that case b/c that's exactly when execution is slower in matching host/target arch case, and the case you want to fix. That would not make things worse for KVM case.

Reply:
If we add check for "-accel tcg", then we can also make it arch-agnostic (not specific for arm64).

Reply:
The difficulty here (as I just figured out) is that "-accel tcg" is not necessarily part of the manager config.
In our case, it is only appended to the command line by vm/qemu/qemu.go based on the arch.

Reply:
Most of that should be handled by the existing `runtime.GOARCH != cfg.SysTarget.VMArch` case.
Either way, your change won't make any of the existing cases worse, it will just improve some cases (that we care about).